### PR TITLE
Modify default label

### PR DESCRIPTION
**Pull request by jameesjohn**
## Explanation

This PR modifies the default label to be added when authors are new contributors.
## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

File: lib/checkPullRequestLabels.js [hunk 1]
```diff
@@ -222,15 +221,31 @@ module.exports.checkChangelogLabel = async function(context) {
...
     });
     await context.github.issues.addLabels(labelParams);
-
+    const reviewers = pullRequest.requested_reviewers.map(
+      (reviewer) => reviewer.login
+    );
+    let commentBody = '';
+    if (reviewers.length === 1) {
+      reviewersToPing = reviewers[0];
+      commentBody =
```
Comment (ankita240796) [inline lib/checkPullRequestLabels.js:233]: Should we assign the reviewers too?

**Review by ankita240796 — CHANGES_REQUESTED** [review]
Just one doubt, PTAL. Thanks @jameesjohn!

File: lib/checkPullRequestLabels.js (hunk 1 above)
Comment (jameesjohn) [reply lib/checkPullRequestLabels.js:234]: Done.

**Review by ankita240796 — APPROVED** [review]
LGTM, Thanks @jameesjohn!

**Pull request merged**